types/src/transaction/authenticator.rs

```diff
@@ -737,16 +802,353 @@ impl fmt::Display for AuthenticationKey {
     }
 }
 
+#[derive(Clone, Debug, Eq, Hash, PartialEq, Serialize, Deserialize)]
+pub struct MultiKeyAuthenticator {
+    public_keys: MultiKey,
+    signatures: Vec<AnySignature>,
+    signatures_bitmap: aptos_bitvec::BitVec,
+}
```

Comment:
What if it was

```rust
pub struct MultiKeyAuthenticator {
    public_keys: Vec<AnyKey>,
    signatures: Vec<AnySignature>,
    signatures_required: u8,
    signatures_bitmap: aptos_bitvec::BitVec,
}
```

Reply:
I had that originally, I found that you want MultiKey for the other places that take it... such as AuthenticationKey.